src/pybind/cephfs.py

```diff
@@ -82,6 +83,12 @@ class cephfs_statvfs(Structure):
                 ("f_flag", c_uint),
                 ("f_namemax", c_uint)]
 
+class cephfs_dirent(Structure):
+    _fields_ = [("d_ino", c_long),
+                ("d_off", c_ulong),
```

Comment:
Yeah, I'm just thinking what we want to expose and what we don't need to. Maybe we just follow C-style function like currently we do. Or totally wrapping the walking directory.